noxfile.py

```diff
@@ -22,7 +22,13 @@ def check(session, reuse_venv=True):
 @nox.session(python=["3.7", "3.8", "3.9", "3.10"])
 def test(session, reuse_venv=True):
     session.install(
-        "pytest", "pytest-cov", "rich", "icecream", "prompt_toolkit", "sqlparse"
+        "pytest",
+        "pytest-cov",
+        "rich",
+        "icecream",
+        "prompt_toolkit",
+        "sqlparse",
+        "IPython",
```

Comment:
For the latest version of Ipython, there is a bug that is about to get reverted.
```suggestion
        "IPython==8.0.0",
```